### PR TITLE
Dynamically load libsensors at runtime

### PR DESCRIPTION
**Pull request by cgzones**
Suggested by @eworm-de

### CODE REVIEW

**Comment by eworm-de** [comment]
Oh, that was fast. 😊
About to test this...

**Comment by eworm-de** [comment]
Works for me.

However I would not mind having libsensors and its development files as a build dependency. That should save some extra code.

File: linux/LibSensors.c [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+dlfailure:
+   if (dlopenHandle) {
+      dlclose(dlopenHandle);
+      dlopenHandle = NULL;
+   }
+   return -1;
+}
+
+void LibSensors_cleanup(void) {
+    if (dlopenHandle) {
```
Comment (BenBE) [inline linux/LibSensors.c:60]: Indentation

Comment (cgzones) [reply linux/LibSensors.c]: thanks, fixed

**Pull request merged**